Week2/num8.js

```diff
@@ -0,0 +1,44 @@
+// 8
+
+// 8.1
+let fooString = "100";
+let fooNumber = 100 ;
+let fooArray = [1, 2, 3];
+let fooX ;
+// 8.2
+console.log("the value of my variable String is: " + fooString)
```

Comment:
I would recommend always adding ';' at the end of a line of code (or function)! Because it might lead to some silly mistakes later on